lib/shopify_app/login_protection.rb

```diff
@@ -26,6 +26,7 @@ def shop_session
 
     def login_again_if_different_shop
       if shop_session && params[:shop] && params[:shop].is_a?(String) && shop_session.url != params[:shop]
+        session[:shopify] = nil
```

Comment:
lol well I guess I should be careful what I ask for! 
It's not so terrible, but I didn't understand since in code this can be retrieved. Understand now it's mostly for when session gets dumped blindly by something like errbit :+1: